dev-docs/bidders/deltaprojects.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: bidder
+title: Delta Projects
+description: Prebid Delta Projects Bidder Adapter
+biddercode: deltaprojects
+gdpr_supported: true
+gvl_id: 209
+media_types: banner
+pbjs: true
+---
```

Comment:
Could you add the information on `safeframes_ok` ? 

```
safeframes_ok: true
```

means your bidder works with safe frames. `false` would mean you require friendly iframes.

Also your adapter will support the floor price module. So you may add

```
floors_supported: true
```

as well